src/app/core/init.ts

```diff
@@ -56,11 +62,9 @@ export type ConnectionInterchangeV1 = {
     entity: string;
     source: any;
     sinks: any;
-    paths?: Array<Array<Point>>;
+    paths: Array<Array<Point>>;
```

Comment:
We need to modify this to match the 1.2 standard `ConnectionPath` (https://github.com/CIDARLAB/Parchmint/issues/11) and have a typedef set connectly